docs/proxy.md

```diff
@@ -17,7 +18,18 @@ cf target -s <APP_SPACE>
 cf add-network-policy <APP_NAME> <PROXY_APP_NAME> --protocol tcp --port 61443 -s <PROXY_SPACE>
 ```
 
-You must get the `vars.challenge-proxy.yml` file from a developer on the team.
+You must get the `vars.challenge-proxy.yml` file from a developer on the team for the current username and password values and the list of allowed connections.
```

Comment:
should we keep this file in a shared g-drive or something?

Reply:
Good call, I placed it there.